lib/chat_api/notes/note.ex

```diff
@@ -0,0 +1,22 @@
+defmodule ChatApi.Notes.Note do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  @primary_key {:id, :binary_id, autogenerate: true}
+  @foreign_key_type :binary_id
+  schema "notes" do
+    field :body, :string
+    field :author_id, :integer
+    field :account_id, :binary_id
+    field :customer_id, :binary_id
+
```

Comment:
can you set up the `belongs_to` relations here too? e.g.

```elixir
    belongs_to(:account, Account)
    belongs_to(:customer, Customer)
    belongs_to(:author, User, foreign_key: :author_id, references: :id, type: :integer)
```

Reply:
(might be nice to add the `has_many` relation to the `ChatApi.Customers.Customer` module in `customer.ex` as well)